pkgs/by-name/ij/ijq/package.nix

```diff
@@ -1,21 +1,37 @@
-{ buildGoModule, fetchFromSourcehut, lib, jq, installShellFiles, makeWrapper, scdoc }:
+{
+  buildGoModule,
+  fetchFromSourcehut,
+  lib,
+  jq,
+  installShellFiles,
+  makeWrapper,
+  scdoc,
+}:
 
 buildGoModule rec {
   pname = "ijq";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromSourcehut {
     owner = "~gpanders";
     repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21